tokio-console/src/main.rs

```diff
@@ -34,6 +36,12 @@ async fn main() -> color_eyre::Result<()> {
         return Ok(());
     }
 
+    if let Some(config::OptionalCmd::GenCompletion { shell }) = args.subcmd {
+        let mut app = config::Config::command();
+        generate(shell, &mut app, "tokio-console", &mut std::io::stdout());
+        return Ok(());
+    }
```

Comment:
style, take it or leave it: now that we have multiple subcommands, we might want to consider collapsing this and the `if` block for the `GenConfig` subcommand into a `match`, like
```rust
match args.subcmd {
    Some(config::OptionalCmd::GenConfig) => // ...
    Some(config::OptionalCmd::GenCompletion { shell } => // ...
    None => {},
}
```

Reply:
fixed: https://github.com/tokio-rs/console/pull/336/commits/b51ce10be556e7371889d0a84ed162c03241b6d9